src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -152,6 +181,11 @@ public void update(TeleopInput input) {
 		double updatedTime = timer.get();
 		currentTime = updatedTime;
 		gyroAngle = getHeading();
+		if (currentTime > 1) {
+			System.out.println("gyro: " + gyroAngle);
+			timer.reset();
+			currentTime = 0;
```

Comment:
Can we name the timer something a bit more descriptive? Also why do we need currentTime and updatedTime if we're just reading from the timer anyway?